stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -1,4 +1,26 @@
 ;; The .pox-4 contract
+;; PoX mainnet constants
+;; Min/max number of reward cycles uSTX can be locked for
+(define-constant MIN_POX_REWARD_CYCLES u1)
+(define-constant MAX_POX_REWARD_CYCLES u12)
+
+;; Default length of the PoX registration window, in burnchain blocks.
+(define-constant PREPARE_CYCLE_LENGTH (if is-in-mainnet u100 u50))
+
+;; Default length of the PoX reward cycle, in burnchain blocks.
+(define-constant REWARD_CYCLE_LENGTH (if is-in-mainnet u2100 u1050))
+
+;; Valid values for burnchain address versions.
+;; These correspond to address hash modes in Stacks 2.0.
+(define-constant ADDRESS_VERSION_P2PKH 0x00)
+(define-constant ADDRESS_VERSION_P2SH 0x01)
+(define-constant ADDRESS_VERSION_P2WPKH 0x02)
+(define-constant ADDRESS_VERSION_P2WSH 0x03)
```

Comment:
Per https://github.com/stacks-network/stacks-core/pull/4117#pullrequestreview-1763898214, can you consolidate these `ADDRESS_VERSION_*` constants with the ones below, at line 65?  Also, can you move the remaining constants _beneath_ the existing constants (after line 75) so as to keep the layout of this file consistent with the other `pox`-es?  Thanks!

Reply:
@jcnelson Addressed in https://github.com/stacks-network/stacks-core/pull/4094/commits/174b6b0b6ff311b627281af68d1eba64c2fd6a90